app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,7 @@
   <string name="topic_page">Topic page</string>
   <string name="topic_name">Topic: %s</string>
   <string name="topic">Topic</string>
+  <string name="ongoing_topic_list_activity_title">Ongoing Topic</string>
```

Comment:
The value of this label/title is incorrect.

It should be `Topics in Progress` a mentioned in the issue.

Reply:
https://github.com/oppia/oppia-android/pull/3002/commits/357c0d18b3b26b16c643de271620ec77ef190ad6
Thanks for bringing this to notice. The label is corrected in the above commit.